run.sh

```diff
@@ -72,13 +72,16 @@ podman run \
     --pod cryostat \
     --mount type=tmpfs,target=/opt/cryostat.d/recordings.d \
     --mount type=tmpfs,target=/opt/cryostat.d/templates.d \
+    --mount type=tmpfs,target=/opt/cryostat.d/probes.d \
     --mount type=bind,source="$(dirname $0)/conf",destination=/opt/cryostat.d/conf.d,relabel=shared,bind-propagation=shared \
     --mount type=bind,source="$(dirname $0)/truststore",destination=/truststore,relabel=shared,bind-propagation=shared \
     --mount type=bind,source="$(dirname $0)/certs",destination=/certs,relabel=shared,bind-propagation=shared \
     --mount type=bind,source="$(dirname $0)/clientlib",destination=/clientlib,relabel=shared,bind-propagation=shared \
     -e CRYOSTAT_PLATFORM=$CRYOSTAT_PLATFORM \
     -e CRYOSTAT_DISABLE_SSL=$CRYOSTAT_DISABLE_SSL \
     -e CRYOSTAT_DISABLE_JMX_AUTH=$CRYOSTAT_DISABLE_JMX_AUTH \
+    -e CRYOSTAT_DISABLE_SSL="true" \
+    -e CRYOSTAT_DISABLE_JMX_AUTH="true" \
```

Comment:
It isn't required to disable these, is it? - just easier for testing temporarily?

Reply:
Yeah it's just easier for temporary testing, I'll revert that before officially opening this PR.